windows-installer/fluent-template.conf

```diff
@@ -27,6 +27,8 @@
   disable_retry_limit
   # Use multiple threads for processing.
   num_threads 8
+  # Use gRPC by default to avoid issues with AWS and SSL certs
```

Comment:
Done.

@igorpeshansky What do you mean not ready? Do you mean for ruby? For the logging gem?